editor/src/clj/editor/game_object.clj

```diff
@@ -481,8 +481,9 @@
                                          (contains? tags :embeddable))
                                  ext))
                              (workspace/get-resource-type-map workspace))]
-    (when-let [resource (first (resource-dialog/make workspace project {:ext component-exts :title "Select Component File"}))]
-      (add-referenced-component! go-id resource select-fn))))
+    (when-let [resources (resource-dialog/make workspace project {:ext component-exts :title "Select Component File" :selection :multiple})]
+      (doseq [resource (if (map? resources) (vals resources) resources)]
```

Comment:
Hmm, I don't think the value returned by `resource-dialog/make` can be a map? 🤔

The docstring says it can return either a list of selected items, or `nil` if they clicked Cancel. Am I missing something?

I think we should remove the `map?` check if it can't be a map, since is may be a bit confusing.

Reply:
I was fighting with this bug for an hour or so yesterday. But now, when I run it, it works just fine. Maybe I broke something else and then, when I fixed it, I forgot to reload it. (Mental note: if something goes unexpectedly wrong, just re-run the editor.)

thank you!